src/Form/FormMapper.php

```diff
@@ -134,13 +138,15 @@ public function add($name, $type = null, array $options = [], array $fieldDescri
                 unset($options['help']);
             }
 
-            $this->formBuilder->add($fieldDescription->getName(), $type, $options);
-
             if (null !== $help) {
-                $this->admin->getFormFieldDescription($fieldDescription->getName())->setHelp($help);
+                $this->admin->getFormFieldDescription($name)->setHelp($help);
             }
         }
 
+        if (!isset($fieldDescriptionOptions['role']) || $this->admin->isGranted($fieldDescriptionOptions['role'])) {
+            $this->formBuilder->add($name, $type, $options);
```

Comment:
line #116 already adds it, won't this add it again?

Reply:
Yes, that call was removed. Thank you!